sdk/communication/Azure.Communication.Identity/tests/Azure.Communication.Identity.Tests.csproj

```diff
@@ -12,14 +12,14 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="$(AzureCoreTestFramework)" />
-    <ProjectReference Include="..\..\Azure.Communication.Common\src\Azure.Communication.Common.csproj" />
-    <ProjectReference Include="..\..\Azure.Communication.Common\tests\Azure.Communication.Common.Tests.csproj" />
     <ProjectReference Include="..\src\Azure.Communication.Identity.csproj" />
   </ItemGroup>
 
   <!-- Shared source from Azure.Core -->
   <ItemGroup>
     <Compile Include="$(AzureCoreSharedSources)ConnectionString.cs" LinkBase="Shared" />
+    <Compile Include="..\..\Shared\src\AuthMethod.cs" LinkBase="Shared\Communication.Tests" />
+    <Compile Include="..\..\Shared\src\CommunicationTestEnvironment.cs" LinkBase="Shared\Communication.Tests" />
```

Comment:
We can simply refer those two classes in just Identity.Tests.csproj file and don't need to make any change in Common.Tests.csproj.
Also, I would recommend referencing those files in different <ItemGroup> so that it becomes more clear that we are referencing them from Common Tests Project.

  
<img width="696" alt="MicrosoftTeams-image" src="https://user-images.githubusercontent.com/115225476/225084596-b58808cc-5560-414b-9100-949fe70ddb71.png">

Reply:
what we're trying to achieve here is to move the shared test code outside from Common.Tests.csproj and keep that project solely for the purpose of testing the Common.csproj